src/index.html

```diff
@@ -21,7 +24,377 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+          class="header__img"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#Apple"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Samsung"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Smartphones"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Laptops & Computers"
+              data-qa="nav-hover"
+              class="header__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Gadgets"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Tablets"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Photo"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#Video"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="laptopFoto"
+        />
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina
          <br />
          5K Monoblock (MNED2UA/A)
        </h2>
```
`h1` tag should be used once per page, do consider using `h2-6` for your cards